test/helpers.js

```diff
@@ -56,9 +56,33 @@ testHelpers.validateJson = function(json) {
   assert.ok(json.links instanceof Object, "Response should have a links block");
   assert.ok(!(json.errors instanceof Object), "Response should not have any errors");
   assert.equal(typeof json.links.self, "string", "Response should have a \"self\" link");
+  testHelpers.validatePagination(json);
   return json;
 };
 
+testHelpers.validatePagination = function(json) {
+  if (!json.meta.page) return;
+  if (!(json.data instanceof Array)) return;
```

Comment:
`Array.isArray` seems to be recommend when supported (as in Node's case) since it covers some corner cases that `instanceof Array` doesn't.